Doc/library/importlib.rst

```diff
@@ -551,6 +555,8 @@ ABC hierarchy::
 
       Reads *path* as a binary file and returns the bytes from it.
 
+      If the *path* cannot be handled, this raises an :exc:`OSError`
+      or a :exc:`ValueError` depending on the reason.
```

Comment:
Exceptions are covered in ResourceLoader abstract method documentation above.

```suggestion
```

Reply:
Actually, I duplicated the entry because the ABC marked as deprecated since 3.7. As such, I'm not sure whether the ABC should update its documentation or not...